jetcd-core/pom.xml

```diff
@@ -181,6 +181,18 @@
 
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <ssl.cert.path>${project.basedir}/../etc/cert</ssl.cert.path>
+                        <ssl.cert.authority>etcd-ssl</ssl.cert.authority>
+                        <ssl.cert.endpoints>https://127.0.0.1:42379</ssl.cert.endpoints>
```

Comment:
I don't quite understand what this part is doing. could you elaborate?
also I am not sure of the official pom.xml should have configs that'sspecific to ssl tests.

Reply:
They are here to avoid hard coding in tests, you may have your own instance and certs you want to use but that can be improved.